### PR TITLE
Update CI in README.md

### LINKED ISSUE #12483: Update CI section in README.md
### TODO
- [ ] add torch 1.11
- [ ] add hpu
- [ ] add ipu

It'd be nice to address this before 1.6 release to showcase our integrations.

______________________________________________________________________

#### If you enjoy Lightning, check out our other projects! ⚡

- [**Metrics**](https://github.com/PyTorchLightning/metrics): Machine learning metrics for distributed, scalable PyTorch applications.

- [**Lite**](https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.html): enables pure PyTorch users to scale their existing code on any kind of device while retaining full control over their own loops and optimization logic.

- [**Flash**](https://github.com/PyTorchLightning/lightning-flash): The fastest way to get a Lightning baseline! A collection of tasks for fast prototyping, baselining, fine-tuning, and solving problems with deep learning.

- [**Bolts**](https://github.com/PyTorchLightning/lightning-bolts): Pretrained SOTA Deep Learning models, callbacks, and more for research and production with PyTorch Lightning and PyTorch.

- [**Lightning Transformers**](https://github.com/PyTorchLightning/lightning-transformers): Flexible interface for high-performance research using SOTA Transformers leveraging Pytorch Lightning, Transformers, and Hydra.

cc @borda @rohitgr7 @tchaton @akihironitta

### PR DESCRIPTION
**Pull request by rohitgr7**
## What does this PR do?

Fixes #12483
Closes #12490 
Also a small changelog and minor doc update.

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda @rohitgr7 @carmocca @akihironitta

### CODE REVIEW

**Review by carmocca — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -354,7 +356,7 @@ ______________________________________________________________________
   <img src="docs/source/_static/images/lightning_lite/lite.gif" height="200px" width="600px">
 </div>
 
-In the Lighting 1.5 release, LightningLite now enables you to leverage all the capabilities of PyTorch Lightning Accelerators without any refactoring to your training loop. Check out the
+In the Lightning 1.5 release, LightningLite now enables you to leverage all the capabilities of PyTorch Lightning Accelerators without any refactoring to your training loop. Check out the
```
[the remark this answers is not in the document]
Comment (rohitgr7) [reply README.md:359]: what do you mean?
it was just a typo

[the remark this answers is not in the document]
Comment (Borda) [reply README.md:359]: no, just feel that this was highlight for the previous release but fine to keep it

[the remark this answers is not in the document]
Comment (rohitgr7) [reply README.md:359]: this was added way after the previous release, so we should keep it for some time.

File: CHANGELOG.md [hunk 2]
```diff
@@ -931,9 +931,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Fixed check for horovod module ([#12377](https://github.com/PyTorchLightning/pytorch-lightning/pull/12377))
 
+
 - Fixed logging to loggers with multiple eval dataloaders ([#12454](https://github.com/PyTorchLightning/pytorch-lightning/pull/12454))
 
 
+- Fixed support loading a checkpoint with QAT ([#11346](https://github.com/PyTorchLightning/pytorch-lightning/pull/11346))
```
Comment (awaelchli) [inline CHANGELOG.md:938]:
```suggestion
- Fixed an issue with resuming from a checkpoint trained with QAT ([#11346](https://github.com/PyTorchLightning/pytorch-lightning/pull/11346))

File: README.md (hunk 1 above)
Comment (awaelchli) [inline README.md:359]: Amazing how we find typos on the README after 2+ years.

File: docs/source/common/trainer.rst [hunk 3]
```diff
@@ -696,9 +696,6 @@ See Also:
 gpus
 ^^^^
 
-.. warning:: Setting `Trainer(gpus=x)` is deprecated in v1.6 and will be removed
```
Comment (awaelchli) [inline docs/source/common/trainer.rst:699]: thx! <3

**Review by awaelchli — APPROVED** [review]
Amazing that we still find typos on the README after 2+ years.

File: README.md [hunk 4]
```diff
@@ -63,29 +63,31 @@ Lightning forces the following structure to your code which makes it reusable an
...
+|  Linux py3.7 \[IPUs\*\*\]  |                                                                                                                             -                                                                                                                             | [![Build Status](<https://dev.azure.com/PytorchLightning/pytorch-lightning/_apis/build/status/PL.pytorch-lightning%20(IPUs)?branchName=master>)](https://dev.azure.com/PytorchLightning/pytorch-lightning/_build/latest?definitionId=6&branchName=master) |                                                                                                                             -                                                                                                                             |                                                                                                                 -                                                                                                                  |
```
Comment (akihironitta) [inline README.md:85]:
```suggestion
|    Linux py3.7 \[IPUs\]    |                                                                                                                             -                                                                                                                             | [![Build Status](<https://dev.azure.com/PytorchLightning/pytorch-lightning/_apis/build/status/PL.pytorch-lightning%20(IPUs)?branchName=master>)](https://dev.azure.com/PytorchLightning/pytorch-lightning/_build/latest?definitionId=6&branchName=master) |                                                                                                                             -                                                                                                                             |                                                                                                                 -                                                                                                                  |
```

File: README.md [hunk 5]
```diff
@@ -63,29 +63,31 @@ Lightning forces the following structure to your code which makes it reusable an
...
+|  Linux py3.7 \[HPUs\*\*\]  |                                                                                                                             -                                                                                                                             |                                                                                                                             -                                                                                                                             | [![Build Status](<https://dev.azure.com/PytorchLightning/pytorch-lightning/_apis/build/status/PL.pytorch-lightning%20(HPUs)?branchName=master>)](https://dev.azure.com/PytorchLightning/pytorch-lightning/_build/latest?definitionId=6&branchName=master) |                                                                                                                 -                                                                                                                  |
```
Comment (akihironitta) [inline README.md:86]:
```suggestion
|    Linux py3.7 \[HPUs\]    |                                                                                                                             -                                                                                                                             |                                                                                                                             -                                                                                                                             | [![Build Status](<https://dev.azure.com/PytorchLightning/pytorch-lightning/_apis/build/status/PL.pytorch-lightning%20(HPUs)?branchName=master>)](https://dev.azure.com/PytorchLightning/pytorch-lightning/_build/latest?definitionId=6&branchName=master) |                                                                                                                 -                                                                                                                  |
```

**Review by akihironitta — APPROVED** [review]
Thank you for updating this!

File: README.md [hunk 6]
```diff
@@ -63,29 +63,31 @@ Lightning forces the following structure to your code which makes it reusable an
...
+|  Linux py3.8 (with Conda   |            [![Test](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml/badge.svg?branch=master&event=push)](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml)             |            [![Test](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml/badge.svg?branch=master&event=push)](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml)             |            [![Test](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml/badge.svg?branch=master&event=push)](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml)             | [![Test](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml/badge.svg?branch=master&event=push)](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml) |
```
Comment (akihironitta) [inline README.md:87]:
```suggestion
|  Linux py3.8 (with Conda)  |            [![Test](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml/badge.svg?branch=master&event=push)](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml)             |            [![Test](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml/badge.svg?branch=master&event=push)](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml)             |            [![Test](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml/badge.svg?branch=master&event=push)](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml)             | [![Test](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml/badge.svg?branch=master&event=push)](https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml) |
```

**Review by akihironitta — APPROVED** [review]
Also, to be precise, I think we have conda tests with (py3.9, torch1.11) but not with (py3.8, torch1.11).

File: README.md [hunk 7]
```diff
@@ -63,29 +63,31 @@ Lightning forces the following structure to your code which makes it reusable an
...
+Once you do this, you can train on multiple-GPUs, TPUs, CPUs, IPUs, HPUs and even in 16-bit precision without changing your code!
 
 Get started with our [2 step guide](https://pytorch-lightning.readthedocs.io/en/latest/starter/new-project.html)
 
 ______________________________________________________________________
 
 ## Continuous Integration
 
-Lightning is rigorously tested across multiple GPUs, TPUs CPUs and against major Python and PyTorch versions.
+Lightning is rigorously tested across multiple CPUs, GPUs, TPUs, and HPUs and against major Python and PyTorch versions.
```
Comment (kaushikb11) [inline README.md:74]:
```suggestion
Lightning is rigorously tested across multiple CPUs, GPUs, TPUs, IPUs, and HPUs and against major Python and PyTorch versions.
```

**Review by kaushikb11 — APPROVED** [review]

**Review by akihironitta — APPROVED** [review]
LGTM! Thank you, Rohit!
